### ISSUE TITLE
Missing code review

### ISSUE THREAD

**Issue by trasherdk** [phpvirtualbox/phpvirtualbox#200]
Did someone check this before merge? #186 
```
	            if($('#vboxPane').data('vboxConfig').enableHDFlushConfig) {
	                opts.push({
                        label: 'Ignore Flush Requests',
                        attrib: 'ignoreFlush',
                        runningEnabled: true,
	                });
	            }; // <----- Semicolon ??
	            if($('#vboxPane').data('vboxConfig').enableAdvancedConfig&&vboxMedia.formatSupportsDiscard(ma.medium.format)) {
	                opts[opts.length]={
                        label: 'Support Discard (TRIM)',
                        attrib: 'discard',
	                };
	            }; // <----- Semicolon ??
	            return opts;
```
In [phpvirtualbox.js](https://github.com/phpvirtualbox/phpvirtualbox/blob/develop/js/phpvirtualbox.js#L4761-L4774)

**Comment by h6w** [comment]
Sorry.  My bad.  Fixed now b4dd0ae3ea297cb8472d99bee165734eaed4a146

**Comment by h6w** [comment]
#222
